app/models/concerns/contest_files.rb

```diff
@@ -38,24 +38,20 @@ def refresh_results_pdf
     )
 
     File.delete(results_location) if File.file?(results_location)
-    unless File.directory?(File.dirname(results_location))
-      FileUtils.mkdir_p(File.dirname(results_location))
-    end
+    FileUtils.mkdir_p(File.dirname(results_location)) unless File.directory?(File.dirname(results_location))
     File.open(results_location, 'wb') { |f| f << pdf }
   end
 
-  private
-
-  def reports_location
+  private def reports_location
     Rails.root.join('public', 'contest_files', 'reports', id.to_s).to_s
   end
 
-  def submissions_location
+  private def submissions_location
```

Comment:
Why is this not corrected yet the bottom one is corrected?

Reply:
Huh, that's weird. Fixed.

Reply:
Ok, this is really weird. After I fixed it, it appears as a rubocop offense.

Reply:
This issue is quite weird: it doesn't actually appear as a rubocop warning. Nevertheless, fixed the top 2.